src/Modules/OrchardCore.Commerce.ContentFields/OrchardCore.Commerce.ContentFields.csproj

```diff
@@ -44,15 +44,12 @@
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" />
   </ItemGroup>
 
-  <ItemGroup Condition="Exists($(LombiqNodeJsExtensionsPath))">
-    <ProjectReference Include="$(LombiqNodeJsExtensionsPath)\Lombiq.NodeJs.Extensions.csproj" />
-  </ItemGroup>
-  <ImportGroup Condition="Exists($(LombiqNodeJsExtensionsPath))">
-    <Import Project="$(LombiqNodeJsExtensionsPath)\Lombiq.NodeJs.Extensions.props" />
-    <Import Project="$(LombiqNodeJsExtensionsPath)\Lombiq.NodeJs.Extensions.targets" />
-  </ImportGroup>
-  <ItemGroup Condition="!Exists($(LombiqNodeJsExtensionsPath))">
-    <PackageReference Include="Lombiq.NodeJs.Extensions" />
-  </ItemGroup>
+  <Target Name="Copy JavaScript Assets"
+          AfterTargets="AfterResolveReferences">
+    <ItemGroup>
+      <JavaScriptAssets Include="Assets\Scripts\**\*.js"/>
+    </ItemGroup>
+    <Copy SourceFiles="@(JavaScriptAssets)" DestinationFolder="wwwroot\js\%(RecursiveDir)" SkipUnchangedFiles="true" />
+  </Target>
```

Comment:
Do we really need this? These files can just go directly into the wwwroot folder. For files coming from NPM packages such a copy (from _node_modules_ to _wwwroot_) would be interesting.

Also see the same copy targets in the other projects.

Reply:
Ironically if I do that, then we have to bring back the package.json to manually configure the asset route to be in wwwroot for the linting. Also that won't be a problem here, but it wouldn't fly in projects where you also copy vendor files into wwwroot. Overall I think this is still cleaner and more intentional.

Reply:
There should be a better way. If we cut down on the needless complexity, then putting files into one folder just to copy them into another during build shouldn't be required.

What do you mean by "manually configure the asset route to be in wwwroot for the linting"? We already configure folders paths when invoking the workflow; can't that be utilized?

Reply:
In the workflow we only provide the project paths, not the asset paths inside the project.

Reply:
Yes, but that can be changed. We can also let users configure the wwwroot, and recognize it by looking at the path.